doc/md/versioned-migrations.md

```diff
@@ -108,6 +108,61 @@ If you want to inspect an existing database and compute the diff against your En
 migration mode.
 :::
 
+### A Word on Global Unique IDs
+
+**This section only applies to MySQL users using the [global unique id](migrate.md/#universal-ids) feature.**
+
+When using the global unique ids, Ent allocates a range of `1<<32` integer values for each table. This is done by giving
+the first table an autoincrement starting value of `1`, the second one the starting value `4294967296`, the third one
+`8589934592`, and so on. The order in which the tables receive the starting value is saved in an extra table
+called `ent_types`. With MySQL 5.6 and 5.7, the autoincrement starting value is only saved in memory and re-calculated 
```

Comment:
Maybe add a link to MySQL docs? https://dev.mysql.com/doc/refman/8.0/en/innodb-auto-increment-handling.html

<img width="1159" alt="image" src="https://user-images.githubusercontent.com/7413593/177494198-417697c5-3f77-44db-9146-605f2c1f3760.png">